packages/calypso-e2e/test/browser-helper.test.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { describe, expect, test, beforeAll, afterAll } from '@jest/globals';
+
+/**
+ * Internal dependencies
+ */
+import { getViewportName, getLocale, getViewportSize } from '../src/browser-helper';
+
+let env: NodeJS.ProcessEnv;
+
+beforeAll( function () {
+	env = process.env;
+} );
+
+describe( 'BrowserHelper Tests', function () {
+	describe( `Test: getViewportName`, function () {
+		test.each`
```

Comment:
Personally I prefer to use the array API (https://jestjs.io/docs/api#1-testeachtablename-fn-timeout). The table API suggest the indentation of the columns is important and people may waste time formatting it. The array API doesn't have this problem as `prettier` will auto-format it.

That being said, I really like the `each` approach, combining good and bad inputs!

Reply:
Interestingly, I didn't spend any time formatting this! `wp-prettier` seems to handle the table format as well.